src/dotnet/Program.cs

```diff
@@ -114,6 +114,8 @@ internal static int ProcessArgs(string[] args, ITelemetry telemetryClient)
                 }
                 else
                 {
+                    firstTimeUseSentinelPath = ConfigureDotNetForFirstTimeUse();
```

Comment:
In this case, I take it that yes. If the user has not seen the EULA, I imagine we don't want to collect telemetry from her.